### DISCUSSION TITLE
Is it possible to install multiple versions of a software at the same time and switch between them, similar to scoop reset, nvm use, etc.

### DISCUSSION THREAD

**Discussion by tianlongJ** [chocolatey/choco#3352] [Q&A]
### Checklist

- [X] I have verified this is the correct repository for opening this issue.
- [X] I have verified no other issues exist related to my request.

### Describe The Solution. Why is it needed?

When setting up a development environment, it is sometimes necessary to switch version compilation for different projects.

**Comment by pauby** [comment]
This isn't an issue but a question. Moving to Discussions.
